client/App/pages/LandingPage/components/Header/VerticalCarousel/VerticalCarouselItem/VerticalCarouselItem.tsx

```diff
@@ -0,0 +1,60 @@
+import { MouseEventHandler } from 'react';
+import styles from './VerticalCarouselItem.module.scss';
+
+interface Props {
+  slide: {
+    clientName: string;
+    clientPicture: string | undefined;
+    clientQuestion: string;
+    specialists: { name: string; avatar: string | undefined }[];
+    updateTime: string;
+    place: string;
+  };
+  stopSlide: MouseEventHandler<HTMLDivElement>;
+  startSlide: MouseEventHandler<HTMLDivElement>;
+}
+
+const CarouselItem = ({ slide, stopSlide, startSlide }: Props) => {
+  return (
+    <div className={styles.carouselItem} onMouseEnter={stopSlide} onMouseOut={startSlide}>
+      <div>
+        <div className={styles.avatar}>
+          {slide.clientPicture === undefined ? (
+            <p>M</p>
+          ) : (
+            <img className={styles.avatarImage} src={slide.clientPicture} />
+          )}
+        </div>
+      </div>
+      <div>
+        <p className={styles.text}>
+          Do
+          <b>
+            <pre> {slide.clientName} </pre>
```

Comment:
I think that this tag here is redundant.

Reply:
Why? It's better to add className to `<pre>` and styling in scss file?